processors/baggage/baggagetrace/processor.go

```diff
@@ -11,15 +11,16 @@ import (
 	"go.opentelemetry.io/otel/sdk/trace"
 )
 
-// BaggageKeyPredicate is a function that returns true if the baggage key should be added to the span.
-type BaggageKeyPredicate func(baggageKey string) bool
+// Filter returns true if the baggage member with key should be added to a
+// span.
+type Filter func(key string) bool
 
-// AllowAllBaggageKeys is a BaggageKeyPredicate that allows all baggage keys.
-var AllowAllBaggageKeys = func(string) bool { return true }
+// AllowAllBaggageKeys allows all baggage members to be added to a span.
```

Comment:
Is there any reason why the filter doesn't accept a baggage.Member instead of a key?

Reply:
I don't see why we couldn't do that in general. Though, I would like to keep this PR scoped to just the rename.

We can address that in a follow up PR though.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5813